Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY conjur-cli.gemspec conjur-cli.gemspec
 COPY lib/conjur/version.rb lib/conjur/version.rb
 
 # Make sure only one version of bundler is available
-RUN gem uninstall bundler -i /usr/local/lib/ruby/gems/${RUBY_VERSION}.0 bundler || true && \
-  gem uninstall bundler -aIx && \
+RUN gem uninstall bundler -i /usr/local/lib/ruby/gems/${RUBY_VERSION} bundler || true && \
```

Comment:
why do we have to run `gem uninstall bundler` twice here?

Reply:
Not sure, but it looks like it's once for the currently selected gemset and once for the version-specific gemset. No idea why this is necessary. Could try without the latter, perhaps.